styles/banner.css

```diff
@@ -0,0 +1,40 @@
+.Cart {
+  font-family: sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+p:not(:last-of-type) {
+  margin-bottom: 0.5rem;
+}
+
+.banner {
+  display: flex;
+  padding: 1rem;
+}
+
+.banner_container {
+  width: auto;
+  font-family: sans-serif;
+  display: inline-block;
+  margin-right: 30px;
+  justify-content: center;
+}
+
+
+.banner--danger {
+  background-color: pink;
+}
+
+.banner--warning {
+  background-color:#FFE7D6;
+}
+.banner_icon   {
```

Comment:
The EDS icon is not 2 rem, it is 24px. (With the default size that is). We shouldn't add an explicit width and height here as this causes spacing issues.